tests/profiler/alice_user_workflow_verbose_profiler_config.yml

```diff
@@ -80,7 +80,8 @@ rules:
         module_name: great_expectations.profiler.expectation_configuration_builder.default_expectation_configuration_builder
         column: $domain.domain_kwargs.column
         min_value: $variables.min_timestamp
-        max_value: $my_max_event_ts.parameter.max_value
+      #  max_value: $my_max_event_ts.parameter.max_value
```

Comment:
I think we probably will want to change this to something between these two options